### ISSUE TITLE
bug: LinkWithPhoneNumber

### ISSUE THREAD

**Issue by mendo96** [capawesome-team/capacitor-firebase#318]
**Plugin(s):**
Firebase Authentication function => LinkWithPhoneNumber

**Platform(s):**
Ios,Android

**Current behavior:**
Hey guys, I want to connect an existing account with a phone number. To do this, I call the LinkWithPhoneNumber function. Here the problem already starts: the interface or the result of the method is 'verificationId', but on the method itself there is a completely different interface. I can only enter the PhoneNumber as a parameter, nothing else.

**Expected behavior:**
Normally, the workflow should be the same as for the SignInWithPhoneNumber function: User enters his phone number, then enters the SMS OTP code in another window. With the LinkWithPhoneNumber method, I also get a VerificationId, but I cannot enter this in the method in the parameter, because I already said that this expects a completely different parameter.

**Comment by jakubforman** [comment]
ℹ️ This fix help only if you linting your code and have problem with return interface/type from method.

For get `verificationId` you need:

```typescript
// @ts-ignore
const { verificationId } = await FirebaseAuthentication.signInWithPhoneNumber()
// @ts-ignore
const {verificationId } = await await FirebaseAuthentication.linkWithPhoneNumber()
```

I know. `@ts-ignore` is not good option, but if you have strict lint you need it for now.

**Comment by mendo96** [comment]
Thank you @jakubforman, that did work for me ! :)

**Comment by robingenz** [comment]
This was fixed in #358 and will be published with the next release.
